data/src/main/kotlin/com/vultisig/wallet/data/repositories/VaultRepository.kt

```diff
@@ -94,15 +98,15 @@ internal class VaultRepositoryImpl @Inject constructor(
     }
 
     override suspend fun deleteTokenFromVault(vaultId: String, tokenId: String) {
-        vaultDao.deleteTokenFromVault(vaultId, tokenId)
+        vaultDao.disableTokenFromVault(vaultId, tokenId)
     }
 
     override suspend fun deleteChainFromVault(vaultId: String, chain: Chain) {
         vaultDao.deleteChainFromVault(vaultId, chain.id)
```

Comment:
@aminsato , can you please remove all the disabled coins belongs to the removed chain?  when user remove a chain from their vault
In this way , when user re-enable the chain , we can re-discovery the tokens on the chain